Scripts/travis.sh

```diff
@@ -3,4 +3,4 @@ git submodule sync && git submodule update --init --recursive
 brew update
 brew outdated swiftlint || brew upgrade swiftlint
 bundle install
-bundle exec fastlane test project_name:$1
\ No newline at end of file
+bundle exec fastlane test project_name:$1 scheme:$2
```

Comment:
Are you sure? Did you test it? As I would expect that this one would fail if `$2` is not passed

Reply:
It just passes null and it all works out ✨